resources/channel.sub.js

```diff
@@ -703,81 +733,119 @@
          * ``null``.
          */
         delete() {
-            if (remoteObjectsById.has(this.objectId)) {
-                remoteObjectsById.delete(this.objectId);
-            }
+            remoteObjectsById.delete(this.objectId);
         }
     }
 
     self.RemoteObject = RemoteObject;
 
-    function serialize(obj) {
-        const stack = [{item: obj}];
-        let serialized = null;
+    /**
+     * Serialize an object as a JSON-compatible representation.
+     *
+     * The format used is similar (but not identical to)
+     * `WebDriver-BiDi
+     * <https://w3c.github.io/webdriver-bidi/#data-types-protocolValue>`_.
+     *
+     * Each item to be serialized can have the following fields:
+     *
+     * type - The name of the type being represented e.g. "string", or
+     *  "map". For primitives this matches ``typeof``, but for
+     *  ``object`` types that have particular support in the protocol
+     *  e.g. arrays and maps, it is a custom value.
+     *
+     * value - A serialized representation of the object value. For
+     * container types this is a JSON container (i.e. an object or an
+     * array) containing a serialized representation of the child
+     * values.
+     *
+     * objectId - An integer if, used to handle object graphs. Where
```

Comment:
I'm not sure if a more words were intended, but removing a word works:

```suggestion
     * objectId - An integer used to handle object graphs. Where
```